crawlers/mooncrawl/mooncrawl/state_crawler/Multicall2.py

```diff
@@ -0,0 +1,487 @@
+    if args.gas_price is not None:
+        transaction_config["gas_price"] = args.gas_price
+    if args.max_fee_per_gas is not None:
+        transaction_config["max_fee"] = args.max_fee_per_gas
+    if args.max_priority_fee_per_gas is not None:
+        transaction_config["priority_fee"] = args.max_priority_fee_per_gas
+    if args.confirmations is not None:
+        transaction_config["required_confs"] = args.confirmations
+    if args.nonce is not None:
+        transaction_config["nonce"] = args.nonce
+    return transaction_config
+
+
+def add_default_arguments(parser: argparse.ArgumentParser, transact: bool) -> None:
+    parser.add_argument(
+        "--network", required=True, help="Name of brownie network to connect to"
+    )
+    parser.add_argument(
+        "--address", required=False, help="Address of deployed contract to connect to"
+    )
+    if not transact:
+        parser.add_argument(
+            "--block-number",
+            required=False,
+            type=int,
+            help="Call at the given block number, defaults to latest",
+        )
+        return
+    parser.add_argument(
+        "--sender", required=True, help="Path to keystore file for transaction sender"
+    )
+    parser.add_argument(
+        "--password",
+        required=False,
+        help="Password to keystore file (if you do not provide it, you will be prompted for it)",
+    )
+    parser.add_argument(
+        "--gas-price", default=None, help="Gas price at which to submit transaction"
+    )
+    parser.add_argument(
+        "--max-fee-per-gas",
+        default=None,
+        help="Max fee per gas for EIP1559 transactions",
+    )
+    parser.add_argument(
+        "--max-priority-fee-per-gas",
+        default=None,
+        help="Max priority fee per gas for EIP1559 transactions",
+    )
+    parser.add_argument(
+        "--confirmations",
+        type=int,
+        default=None,
+        help="Number of confirmations to await before considering a transaction completed",
+    )
+    parser.add_argument(
+        "--nonce", type=int, default=None, help="Nonce for the transaction (optional)"
+    )
+    parser.add_argument(
+        "--value", default=None, help="Value of the transaction in wei(optional)"
+    )
+    parser.add_argument("--verbose", action="store_true", help="Print verbose output")
+
+
+def handle_deploy(args: argparse.Namespace) -> None:
+    network.connect(args.network)
+    transaction_config = get_transaction_config(args)
+    contract = Multicall2(None)
+    result = contract.deploy(transaction_config=transaction_config)
+    print(result)
+    if args.verbose:
+        print(result.info())
+
+
+def handle_verify_contract(args: argparse.Namespace) -> None:
+    network.connect(args.network)
+    contract = Multicall2(args.address)
+    result = contract.verify_contract()
+    print(result)
+
+
+def handle_aggregate(args: argparse.Namespace) -> None:
+    network.connect(args.network)
```

Comment:
We need to apply to work with https://github.com/bugout-dev/moonstream/blob/e1477643b9f12d82402fa6d5239e5c6868999621/crawlers/mooncrawl/mooncrawl/blockchain.py#L46
Because there is no brownie at production